spec/marqeta/transaction_spec.rb

```diff
@@ -14,50 +15,61 @@
                 'token' => 'token1',
                 'state' => Marqeta::Transaction::PENDING_STATE,
                 'user_token' => 'user_token1',
-                'amount' => 1000
+                'amount' => 1000,
+                'created_time' => '2018-01-01T00:00:00Z',
+                'card_acceptor' => {
+                  'name' => 'Marqeta'
+                }
               },
               {
                 'token' => 'token2',
                 'state' => 'DECLINED',
                 'user_token' => 'user_token2',
-                'amount' => 2000
+                'amount' => 2000,
+                'created_time' => '2018-01-02T00:00:00Z',
+                'card_acceptor' => {
+                  'name' => 'Financeit'
```

Comment:
Minor, but since this is a public gem, maybe this should be something more generic?

Reply:
I was kind of thinking like it as an Easter Egg. It is off of the Financeit Github account.

But if you still think I should change it then NBD for sure.